bots/pr/src/test/java/org/openjdk/skara/bots/pr/CommandTests.java

```diff
@@ -103,4 +103,51 @@ void helpCommand(TestInfo testInfo) throws IOException {
             assertEquals(1, error);
         }
     }
+
+    @Test
+    void selfCommand(TestInfo testInfo) throws IOException {
+        try (var credentials = new HostCredentials(testInfo);
+             var tempFolder = new TemporaryDirectory()) {
+            var author = credentials.getHostedRepository();
+            var integrator = credentials.getHostedRepository();
+
+            var censusBuilder = credentials.getCensusBuilder()
+                                           .addAuthor(author.forge().currentUser().id());
+            var mergeBot = PullRequestBot.newBuilder().repo(integrator).censusRepo(censusBuilder.build()).build();
+
+            // Populate the projects repository
+            var localRepo = CheckableRepository.init(tempFolder.path(), author.repositoryType());
+            var masterHash = localRepo.resolve("master").orElseThrow();
+            assertFalse(CheckableRepository.hasBeenEdited(localRepo));
+            localRepo.push(masterHash, author.url(), "master", true);
+
+            // Make a change with a corresponding PR
+            var editHash = CheckableRepository.appendAndCommit(localRepo);
+            localRepo.push(editHash, author.url(), "refs/heads/edit", true);
+            var pr = credentials.createPullRequest(author, "master", "edit", "This is a pull request");
+
+            // Issue an command using the bot account
+            var botPr = integrator.pullRequest(pr.id());
+            botPr.addComment("/help");
+            TestBotRunner.runPeriodicItems(mergeBot);
+
+            // The bot should not reply
+            assertEquals(1, pr.comments().size());
+            TestBotRunner.runPeriodicItems(mergeBot);
+            assertEquals(1, pr.comments().size());
```

Comment:
Should to `runPeriodicItems` call right after adding the "/help" comment be there? Asking because it looks like you want to check the number of comments before and after running the bot below that (but at that point the bot has already run).

Reply:
Right, I put the asserts around the bot execution to make it clear that the number doesn't change, but the first invocation certainly obfuscates that.. Will remove it.